File: engine/engine-model/src/main/scala/tools/packet/cashshop/SendMapleLifeError.scala
package tools.packet.cashshop

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class SendMapleLifeError(private var _code: Int) extends PacketInput {
  def code: Int = _code

  override def opcode(): SendOpcode = SendOpcode.MAPLELIFE_ERROR
}

File: engine/engine-model/src/main/java/tools/packet/cashshop/SendMapleLifeError.java
package tools.packet.cashshop;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record SendMapleLifeError(Integer code) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.MAPLE_LIFE_ERROR;
   }
}